Tools/AP_Bootloader/board_types.txt

```diff
@@ -261,6 +261,7 @@ AP_HW_CUBEORANGE_PERIPH              1400
 AP_HW_CUBEBLACK_PERIPH               1401
 AP_HW_PIXRACER_PERIPH                1402
 AP_HW_SWBOOMBOARD_PERIPH             1403
+AP_HW_EchoPilotAI                    1404
```

Comment:
apologies, this is failing CI with:
  ValueError: Duplicate ID 1404 in file for (AP_HW_EchoPilotAI) and (AP_HW_VIMDRONES_FLOW)